test/files/run/t6333.scala

```diff
@@ -23,6 +23,9 @@ object Test extends App {
     Try(expr) filter (_ => true)
     Try(expr) recover { case _ => fail }
     Try(expr).failed
+    Try(expr).contains(expr)
+    Try(expr).exists(_ => false)
+    Try(expr).exists(_ == expr)
```

Comment:
This adds to a minimized test case for a previous defect.  I suppose it's a behavior that the other Try test suites don't capture, so maybe it's ok to hijack.

Reply:
I don't ... I'm missing something?

Reply:
t6333 is a reference to an issue number, you've added your test cases to it.

Reply:
Yes, that's what I should have said.  Sorry for the confusing comment.

Reply:
reverted